cg/meta/deliver.py

```diff
@@ -46,6 +46,7 @@ def __init__(
         self.ticket_id: str = ""
         self.dry_run = False
         self.delivery_type: str = delivery_type
+        self.skip_missing_bundle = False
```

Comment:
```suggestion
        self.skip_missing_bundle = self.delivery_type in constants.SKIP_MISSING
```

And then remove the if stuff from L63-64